lib/OpenQA/Utils.pm

```diff
@@ -1078,4 +1079,12 @@ sub any_array_item_contained_by_hash {
 
 sub base_host { Mojo::URL->new($_[0])->host || $_[0] }
 
+sub job_label {
+    my ($job) = @_;
+
+    my $test    = $job->TEST;
+    my $machine = $job->MACHINE;
+    return $machine ? "$test\@$machine" : $test;
+}
```

Comment:
This function should be defined within the DBIx result class. I don't get why you want to move it here.